data-academy.qmd

```diff
@@ -0,0 +1,71 @@
+
+[Sign up for spring 2025] **by March 24, 2025** at 10am PT ⬢ Sample email to share announcement **\[LINK\]**
+
+## What is the Data Academy?
+
+**What:** You will learn R through lessons on the [dataquest.io](http://dataquest.io) platform. The Data Academy is **asynchronous** so you can complete the lessons on your own schedule, and also **cohort-based** so you’ll get to know your colleagues and have accountability. The Openscapes team will provide support through **synchronous help desk hours** and a **dedicated Google Space** for help. A training certificate will be issued for each module completed.
+
+**Expected time commitment**: 2-hour chunks of learning time, plus 1 hour of optional office hours support, weekly for 3 months.
+
+**When**: April through June, 2025.
+
+-   Mondays: Week’s lesson reminder received via email. Complete lesson on your own schedule.
+-   Wednesdays: Help desk with the Openscapes team and other participants (Optional, via Zoom, 12-1 pm EDT)
+
+**Who**: FTEs and Affiliates from any Fisheries office are welcome to sign up and there is no cost to staff or your center/office. The 40 available spots will be distributed across centers and offices.
+
+**Cost:** Free; this opportunity is supported by NOAA funding to Openscapes.
+
+## Curriculum
+
+Completion of the following gives a certificate.
+
+**TODO** Something about dataquest nomenclature and ours - career paths, courses, lessons, module, skills, office hours vs help desk.
+
+-   [R Basicis for Data Analysis](https://www.dataquest.io/path/r-basics-for-data-analysis/)
+-   [Introduction to Data Analysis in R](https://www.dataquest.io/course/intro-to-r-rewrite/)
+-   [Data Structures in R](https://www.dataquest.io/course/datastructure-in-r-rewrite/)
+-   [Control Flow, Iteration, and Functions in R](https://www.dataquest.io/course/intermediate-r/)
+-   [Specialized Data Processing in R](https://www.dataquest.io/course/intermediate-r-part-two/)
+-   [Data Visualization with R](https://www.dataquest.io/path/data-visualization-with-r/)
+
+During the Data Academy, you will have access to the full dataquest.io library of courses in addition to the R Curriculum. Once you have completed the curriculum for your certificate, we recommend the following:
+
+-   [Introduction to Data Cleaning in R](https://www.dataquest.io/course/r-data-viz-2/)
+-   [Introduction to Git and Version Control](https://www.dataquest.io/course/git-and-vcs/)
+-   [Querying Databases with SQL and R](https://www.dataquest.io/course/querying-databases-with-r/)
+
+## FAQ
+
+**Who can participate?**
+
+FTEs & Affiliates. We are trying to get representation from all NOAA Fisheries science centers, regional offices, and strategic initiatives.
+
+**What’s the minimum I need to know?**
+
+Absolute beginners are encouraged to apply. No prerequisites.
```

Comment:
Is it too obvious to state that people should have data-related activities as part of their work, and be comfortable working with data in other contexts (e.g., excel, databases, GIS, other languages such as Python, Matlab, etc?)

Reply:
good idea!

Reply:
added a line for this